packages/core/src/components/overlay/overlay.tsx

```diff
@@ -229,10 +229,22 @@ export class Overlay extends AbstractPureComponent2<OverlayProps, IOverlayState>
     // an HTMLElement that contains the backdrop and any children, to query for focus target
     public containerElement: HTMLElement | null = null;
 
+    private startFocusTrapElement: HTMLDivElement | null = null;
```

Comment:
would be nice to leave a brief `// comment` here about this member's purpose. same for `endFocusTrapElement`.